src/EPR.Accreditation.API/Services/AccreditationService.cs

```diff
@@ -91,7 +91,7 @@ public async Task<IEnumerable<AccreditationTaskProgress>> GetTaskProgress(Guid e
         }
 
         public async Task AddFile(
-            Guid id, 
+            Guid id,
```

Comment:
Becauyse we're in the service layer here, we can call "Id" the more appropriate "externalId" (Do the same on line 156)

Reply:
Amended